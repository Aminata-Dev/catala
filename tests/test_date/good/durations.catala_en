## Article

```catala
declaration scope A:
  output x content date
  output y content date
  output z content boolean
  output z2 content boolean
  output z3 content duration
  output d content decimal
  output d2 content decimal
  output m content duration
  output m2 content duration

scope A:
  definition x equals |2019-01-01|
  definition y equals |2002-09-30|

  definition z equals y +@ (x -@ y) = x
  definition z3 equals x -@ y
  definition z2 equals z3 = 5937 day

  definition d equals z3 /^ z3
  definition d2 equals z3 /^ 2 day

  definition m equals z3 *^ 2

  definition m2 equals (2 month) *^ 3
```

```catala-test {id="A.Interpret"}
catala Interpret -s A
```
