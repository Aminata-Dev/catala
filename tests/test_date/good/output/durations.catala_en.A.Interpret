[RESULT] Computation successful! Results:
[RESULT] m = [0 years, 0 months, 11874 days]
[RESULT] m2 = [0 years, 6 months, 0 days]
[RESULT] x = 2019-01-01
[RESULT] y = 2002-09-30
[RESULT] z = true
[RESULT] z2 = true
[RESULT] z3 = [0 years, 0 months, 5937 days]
