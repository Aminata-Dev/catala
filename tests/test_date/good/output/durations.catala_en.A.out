[RESULT] Computation successful! Results:
[RESULT] d = 1.
[RESULT] x = 2019-01-01
[RESULT] y = 2002-09-30
[RESULT] z = true
[RESULT] z2 = true
[RESULT] z3 = 5937 days
